Packs/AzureDataExplorer/Integrations/AzureDataExplorer/AzureDataExplorer.py

```diff
@@ -0,0 +1,573 @@
+
+        return self.management_query_request(database_name, mgmt_query)
+
+    def running_search_query_delete_request(self, database_name: str, client_activity_id: str,
+                                            reason: str) -> Dict[str, Any]:
+        """
+        Starts a best-effort attempt to cancel a specific running search query
+        on the given database.
+
+        Args:
+            database_name (str): The name of the database to see the completed queries on.
+            client_activity_id (str):  Client specified identity of the request.
+            reason (str): The reason for the cancellation.
+        Returns:
+            Dict[str, Any]: API response from Azure.
+        """
+        cancel_running_query = f".cancel query '{client_activity_id}'"
+
+        if reason:
+            cancel_running_query += f" with ( reason = '{reason}' )"
+        return self.management_query_request(database_name, cancel_running_query)
+
+    def management_query_request(self, database_name: str, mgmt_query: str) -> Dict[str, Any]:
+        """
+            API call method for management query endpoint.
+            Each requests that uses management query endpoint uses this method.
+           Args:
+               database_name (str): The name of the database to see the completed queries on.
+               mgmt_query (str):  Client specified identity of the request.
+           Returns:
+               Dict[str, Any]: API response from Azure.
+           """
+        data = retrieve_common_request_body(database_name, mgmt_query)
+        response = self.http_request("POST", url_suffix="/v1/rest/mgmt", data=data)
+        return response
+
+
+def search_query_execute_command(client: DataExplorerClient, args: Dict[str, Any]) -> CommandResults:
+    """
+    Execute search query command.
+    Args:
+        client (DataExplorerClient): Azure Data Explorer API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+    Returns:
+        CommandResults: Command results with raw response, outputs and readable outputs.
+
+    """
+    query = str(args['query'])
+    database_name = str(args['database_name'])
+    timeout = arg_to_number(args.get('timeout', '5'))
+    if timeout < 1 or timeout > 60:
+        raise ValueError("Timeout argument should be a number between 1 to 60.")
+
+    client_activity_id = f"{client.client_activity_prefix};{uuid.uuid4()}"
+    response = client.search_query_execute_request(database_name, query, timeout, client_activity_id)
+    response_kusto_dataset = KustoResponseDataSetV1(response)
+    primary_results = convert_kusto_response_to_dict(response_kusto_dataset)
+    outputs = {
+        'Query': query,
+        'ClientActivityID': client_activity_id,
+        'PrimaryResults': primary_results
+    }
+    readable_output = tableToMarkdown(
+        f'Results of executing search query with client activity ID: {client_activity_id}',
+        primary_results, headerTransform=pascalToSpace)
```

Comment:
I don't think that this is relevant since the "search-query-execute" will run against any Database, so we can't anticipate the order of the results, the names of the columns or their importance. In addition, the user himself decides what query to execute so he owns the choice which columns will be returned and their order.